.github/workflows/deploy-gcp-tests.yml

```diff
@@ -206,7 +210,7 @@ jobs:
           LOCAL_STATE_VERSION=$(grep -oE "DATABASE_FORMAT_VERSION: .* [0-9]+" "$GITHUB_WORKSPACE/zebra-state/src/constants.rs" | grep -oE "[0-9]+" | tail -n1)
           echo "LOCAL_STATE_VERSION: $LOCAL_STATE_VERSION"
 
-          ZEBRA_STATE_DISK=$(gcloud compute images list --filter="name~${{ inputs.disk_prefix }} AND name~-${{ inputs.disk_suffix }}" --format="value(NAME)" --sort-by=~creationTimestamp --limit=1)
+          ZEBRA_STATE_DISK=$(gcloud compute images list --filter="name~zebrad-cache-main AND name~-${{ inputs.disk_suffix }}" --format="value(NAME)" --sort-by=~creationTimestamp --limit=1)
```

Comment:
I fixed disk selection in PR #4385 based on ticket #4153, can you please revert this change?

Reply:
#4269 might add more logic to this pieces of code. I'd prefer to finish this PR and #4269 (critical path) and then focus on #4385 (which seems good so far.

Or merge #4385 even before this one, so we reduce the amount of refactors needed after #4269

Reply:
I'll try to get #4385 done now.

But if it doesn't work today, let's go ahead with this PR and #4269, and then fix up #4385 later.

Reply:
I split the required changes out into PR #4391.

So I'd like to merge that PR first, then merge this one.